### PR TITLE
add custom input handling; validate sentences

### PR DESCRIPTION
**Pull request by SylvanSign**
My Magnum Opus

### CODE REVIEW

**Comment by SylvanSign** [comment]
you can test it out here (need at least 3 players/tabs to start the game)  http://54.174.101.223/

File: public/index.html [hunk 1]
```diff
@@ -7,12 +7,15 @@
   <title>Are you alive?</title>
```
Comment (ZakMiller) [inline public/index.html:7]: I miss the ">"

File: public/scripts/customInput.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+  currentWord.textContent += key
+  classifyWord()
+}
+
+function handleSpace() {
+  if (currentWord.textContent === '') {
+    return // do nothing if no word to add
+  }
+  wordSet.add(currentWord.textContent)
+  currentWord = addListItemAsPenultimateChild(words, ['word'])
```
Comment (ZakMiller) [inline public/scripts/customInput.js:106]: So we aren't validating whether words are actual words yet. That's a different story I guess?

File: public/index.html (hunk 1 above)
Comment (SylvanSign) [reply public/index.html:7]: I can add it back bb

File: public/scripts/customInput.js (hunk 2 above)
Comment (SylvanSign) [reply public/scripts/customInput.js:106]: Yeah, I was getting a bit overwhelmed with the possible features and I want to keep moving forward. One, so I don't block you guys from working on top of the fundamental parts I built, and Two, so we can have smaller PRs more frequently, which is the easier and better way of doing things I think

File: public/index.html [hunk 3]
```diff
@@ -41,12 +46,11 @@
     </div>
 
     <footer>
-      <label class="prompt" for="input">&gt;</label><input id="input" autocomplete="off" />
-      <audio id="errorSound">
-        <source src="airhorn.mp3" type ="audio/mp3">
-      </audio>
+      <p id='wordCounter'></p>
+      </span><span id='customInput'></span>
```
Comment (SylvanSign) [inline public/index.html]: Fix broken tag

**Comment by SylvanSign** [comment]
added a commit to spellcheck words server-side

File: public/index.html (hunk 1 above)
Comment (SylvanSign) [reply public/index.html:7]: I can't figure out how to add it back without it glitching out for me. I might need your help with the CSS

**Comment by SylvanSign** [comment]
added a commit to make the server generate the random word lists and validate the final sentence upon receiving a message from clients

**Comment by SylvanSign** [comment]
pushed a commit to make the server push out the random word lists on its own time. With this change, we can play a very rudimentary version of the game now, with some manual bookkeeping thrown in

**Comment by SylvanSign** [comment]
I want to walk you guys through this PR since there's a lot going on

**Comment by ZakMiller** [comment]
Looks good to me.

**Review by ZakMiller — APPROVED** [review]

**Pull request merged**